docs/before/analysis.md

```diff
@@ -0,0 +1,155 @@
+
+##### Setup the workflow definition in the eLog
+Navigate to the "Workflow definition" tab [https://pswww.slac.stanford.edu/lgbk/lgbk/{experiment}/#lcls_wf_defs]() (see step 2 in the figure above) and click on the "+" button located at the top right of the window. 
+Enter the following information:
+
+- *Name*: the memorable name you wish to call this workflow.
+- *Trigger*: choose whether you will trigger the workflow manually or automatically.
+- *Location*: leave "SLAC" for now. More soon...
+- *Executable*: the path to the `btx` eLog trigger script: `/cds/sw/package/autosfx/btx/scripts/elog_trigger.py`.
+- *Parameters*: the command-line arguments for the `elog_trigger.py` script. See just below for their descriptions:
+
+```
+| Flag  | Meaning                                             | Usage                |
+| -d    | Name of the DAG you wish to use.                    | -d <dag>             |
+| -q    | Queue to use.                                       | -q <queue>           |
+| -n    | Number of cores (soon deprecated)                   | -n <n_cores>         |
+| -c    | Path to the configuration file in YAML format.      | -c <config_file>     |
+```
+
+##### Trigger the workflow from the eLog
+Navigate to the "Workflow control" tab [https://pswww.slac.stanford.edu/lgbk/lgbk/{experiment}/#lcls_wf_jobs]() (see step 3 in the figure above). To trigger a worflow for a given run, select the workflow "Name" from the list in the "Job" column. 
+To trigger it for multiple consecutive runs, click on the train icon at the top right of the window, select your workflow "Name", and enter the first and last run numbers to be processed.
+
+##### Monitor the workflow results from the eLog
+Navigate to the "Summaries" tab [https://pswww.slac.stanford.edu/lgbk/lgbk/{experiment}/#summaries]() (See step 4 in the figure above). 
+In the navigation bar on the left of the window, you should see a list of "samples" and "runs". Click on them to display the results associated with the corresponding "sample" or "run".
+
+### Contributing to the code
+
+The repository can be cloned from this [Github link](https://github.com/lcls-users/btx). The `README.md` describes guidelines for contributing to the code. 
+
+- Contributing a new task
+:    New tasks should be added as separate functions to [tasks.py](https://github.com/lcls-users/btx/blob/main/scripts/tasks.py) and then can be run from the command-line as described above. 
+
+- Contributing new workflows 
+:    To create a new workflow, make sure the tasks you wish to chain in the DAG exist. Then create a new file in the `dags/` subfolder of `btx` (get some inspiration from the [existing ones](https://github.com/lcls-users/btx/tree/main/dags)). The DAG will be available shortly after your PR has been merged in the main `btx` branch.
```

Comment:
Maybe this section can be expanded slightly as the first step in the section above?